core/src/utils/networking.cpp

```diff
@@ -289,7 +288,7 @@ namespace net {
                 }
                 entry.handler(std::move(client), entry.ctx);
             }
-            catch (std::exception e) {
+            catch (const std::exception& e) {
```

Comment:
here too I don't care much about const correctness since thus won't affect performance and just make things less readable